numpy/lib/tests/test_index_tricks.py

```diff
@@ -106,6 +106,9 @@ def test_big_indices(self):
                 np.ravel_multi_index(arr, (41, 7, 120, 36, 2706, 8, 6)),
                 [5627771580, 117259570957])
 
+        # test unravel_index for big indices (issue #9538)
+        assert_raises(ValueError, np.unravel_index, 1, (4294967295, 2147483649))
```

Comment:
nit: the test intent is clearer as `(2**32-1, 2**31+1)`

Reply:
On 32bits this ValueError is actually `ValueError: Maximum allowed dimension exceeded` a bit earlier on. Could get around by using 3 values, but does not really matter anyway.